tasks.py

```diff
@@ -1,11 +1,14 @@
 from invoke import Collection
 
-from environments.surfpol import environment
+from environments.surfpol import create_configuration_and_session
 from elastic.tasks import setup, create_snapshot, load_repository, restore_snapshot
 from deploy import prepare_builds, build, push, deploy
 from test import test
 
 
+environment, session = create_configuration_and_session(use_aws_default_profile=True)
```

Comment:
To clarify. Fabric will use configuration in a very similar way and then sends entire configs to bastion. That way the bastion doesn't need access to secrets, while it can use the passwords occasionally when running commands.